packages/api/helper/googleDrive.js

```diff
@@ -0,0 +1,49 @@
+const { google } = require('googleapis');
+const parentFolderId = '1Uu04G0GGvJVaYE0S9hvc9_6lmtN5XQtQ'
+
+const credentials = require('../env/credentials.json');
```

Comment:
Can this use the .env file instead so all the credentials are in the same place?